helm/kyverno-policies-connectivity/templates/Pod.yaml

```diff
@@ -59,9 +85,9 @@ spec:
                   {{- end }}
                   {{- if $proxy.noProxy }}
                   - name: NO_PROXY
-                    value: {{ join "," $proxy.noProxy }}
+                    value: {{ $proxy.noProxy }}
                   - name: no_proxy
-                    value: {{ join "," $proxy.noProxy }}
```

Comment:
We already provide noProxy as a string like "mydomain.com, myseconddomain.org" . We don't need join here.